pkg/collect/host_subnetavailable.go

```diff
@@ -0,0 +1,287 @@
+package collect
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/debug"
+)
+
+type SubnetStatus string
+
+const (
+	SubnetStatusAvailable     = "a-subnet-is-available"
+	SubnetStatusNoneAvailable = "no-subnet-available"
+)
+
+type SubnetAvailableResult struct {
+	CIDRRangeAlloc string `json:"CIDRRangeAlloc"`
+	DesiredCIDR    int    `json:"desiredCIDR"`
+	// If subnet-available, at least 1 of the DesiredCIDR size is available within CIDRRangeAlloc
+	Status SubnetStatus `json:"status"`
+}
+
+type CollectHostSubnetAvailable struct {
+	hostCollector *troubleshootv1beta2.SubnetAvailable
+	BundlePath    string
+}
+
+func (c *CollectHostSubnetAvailable) Title() string {
+	return hostCollectorTitleOrDefault(c.hostCollector.HostCollectorMeta, "Subnet Available")
+}
+
+func (c *CollectHostSubnetAvailable) IsExcluded() (bool, error) {
+	return isExcluded(c.hostCollector.Exclude)
+}
+
+func (c *CollectHostSubnetAvailable) Collect(progressChan chan<- interface{}) (map[string][]byte, error) {
+	procNetRoute, err := os.ReadFile("/proc/net/route")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to read contents of /proc/net/route")
+	}
+
+	routes, err := parseProcNetRoute(string(procNetRoute))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to parse /proc/net/route")
+	}
+	debug.Printf("Routes: %+v\n", routes)
```

Comment:
This spits out hard to read logs. Some formatting would be good to have. Also this feels like a `V(2)` type of log.

Reply:
I opted to make these `V(3)` instead... sufficient? formatting these feels like I'm reinventing `ip route list` 😆 
In theory you shouldn't need it too often tbh...